README.md

```diff
@@ -6,79 +6,64 @@ transfers made easy<br><br>
 </h3>
 
 
-
 [![CI](https://github.com/astronomer/apache-airflow-provider-transfers/actions/workflows/ci-uto.yaml/badge.svg)](https://github.com/astronomer/apache-airflow-provider-transfers)
 
-The **Universal Transfer Operator** allows data transfers between any supported source and target Datasets in [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, simplifying the users' experience, so they do not need to use specific providers or operators for transfers. The Universal Transfer Operator is maintained by [Astronomer](https://astronomer.io).
+The **Universal Transfer Operator** simplifies how users transfer data from a source to a destination using [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, simplifying the users' experience, so they do not need specific providers or operators. It is maintained by [Astronomer](https://astronomer.io).
 
-This ensures a consistent set of data providers that can read from and write to the dataset. The Universal Transfer
-Operator can use the respective data providers to transfer between a source and a destination. It also takes advantage of any existing fast and
-direct high-speed endpoints, such as Snowflake's built-in ``COPY INTO`` command to load S3 files efficiently into Snowflake.
 
-Universal transfer operator also supports the transfers using third-party platforms like Fivetran.
+## Installation
 
-## How Universal Transfer Operator Works
+```
+pip install apache-airflow-provider-transfers
+```
 
+## How Universal Transfer Operator Works
 
 ![Approach](./docs/images/approach.png)
 
-With universal transfer operator, users can perform data transfers using the following transfer modes:
+With Universal Transfer Operator, users can perform data transfers using the following transfer modes:
 
 1. Non-native
 2. Native
 3. Third-party
 
+
 ### Non-native transfer
 
-When we load a data located in one dataset located in cloud to another dataset located in cloud, internally the steps involved are:
+Non-native transfers rely on transferring the data through the Airflow worker node. Chunking is applied where possible. This method can be suitable for datasets smaller than 2GB, depending on the source and target. The performance of this method is highly dependent upon the worker's memory, disk, processor and network configuration.
```

Comment:
I think we should not add 2GB as we are yet to do benchmarking, and this method can be slow for that size. We can be a little vague here by just saying `smaller dataset`

Reply:
It is essential that we give an idea of size. The term "smaller dataset" is too subjective, and the size matters since we're offering a data transfer tool. We can use our last benchmark as a reference since most of the implementation (except for Fivetran) was brought from the Python SDK, WDYT? Would you feel more comfortable with 1GB, or another value?

The way it is phrased doesn't give a guarantee and is subject to source and target.

Reply:
I'm on the fence about this because those numbers are of native transfers in SDK and we don't have them in UTO right now.